.env.example

```diff
@@ -1,5 +1,16 @@
 ACCESS_TOKEN_SECRET=
 REFRESH_TOKEN_SECRET=
+RESET_PASSWORD_TOKEN_SECRET=
+VERIFY_EMAIL_TOKEN_SECRET=
 MONGO_URI=
 
-NODE_ENV=
\ No newline at end of file
+FRONTEND_BASE_URL=http://localhost:3000
+
+APP_NAME=Issue Solver
+NODE_ENV=development
+SALT=
```

Comment:
salt that is used in bcrypt must be an integer. Value from .env is a string so every use of it need to be parsed to int.